### PR TITLE
Reference materials for loading data

### PR DESCRIPTION
**Pull request by Abby-Wheelis**
After recent struggles remembering how to carry out the different variations of loading data into an opcode on the local server to then work with it in the emulator, I'm documenting them here for future reference. The file that I included references how to:

- load data from the sample files in the server
- load data from gz files
- work with a deployment config on the local server
- make up a valid opcode

Once this is finalized, should I link this anywhere? I could see it being useful linked to server, phone, and/or deploy-configs

### CODE REVIEW

File: docs/dev/front/loading_data_for_emulator.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+`./e-mission-py.bash bin/debug/intake_single_user.py -e [user-0 or updated]`
+
+If you updated to a valid opcode, you can now start the server and log into the devapp with that opcode
+
+## Note on Working with Deployment Configs
+
+There are development configs designed to be used with the local server. They include `dev-emulator-program`, `dev-emulator-study`, and `dev-emulator-timeuse`. 
+
+Sometimes, it is useful to load data into a deployment config for testing, demo, or debugging purposes. If you want to use the local server and a deployment config, you need to take a few extra steps.
```
Comment (shankari) [inline docs/dev/front/loading_data_for_emulator.md:34]: @Abby-Wheelis you meant "development" here, right?

File: docs/dev/front/loading_data_for_emulator.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+## Note on Working with Deployment Configs
+
+There are development configs designed to be used with the local server. They include `dev-emulator-program`, `dev-emulator-study`, and `dev-emulator-timeuse`. 
+
+Sometimes, it is useful to load data into a deployment config for testing, demo, or debugging purposes. If you want to use the local server and a deployment config, you need to take a few extra steps. 
+1. Fork `nrel-openpath-deploy-configs`
+2. Modify the config you want to work on, making sure that there is no `server` defined (if no server is defined, the app will connect to the local server, as desired).
+3. Update the downloadURL in `dynamicConfig` for the version of `e-mission-phone` you are running to `https://raw.githubusercontent.com/[MY-REPO]/nrel-openpath-deploy-configs/[MY-BRANCH]/configs/${label}.nrel-op.json`
+4. Now you can use a development config with the local server
```
Comment (shankari) [inline docs/dev/front/loading_data_for_emulator.md:38]: To be consistent with this?

**Review by shankari — CHANGES_REQUESTED** [review]
Otherwise LGTM!

File: docs/dev/front/loading_data_for_emulator.md (hunk 1 above)
Comment (Abby-Wheelis) [reply docs/dev/front/loading_data_for_emulator.md:34]: I don't think so. I'm talking about if you want to load data into a "deployment" config (like I needed to with `usaid-laos-ev` last week, as opposed to "normal" which would be a "development" config.

File: docs/dev/front/loading_data_for_emulator.md (hunk 2 above)
Comment (Abby-Wheelis) [reply docs/dev/front/loading_data_for_emulator.md:38]: What I meant to do was to have this one say "deployment"

**Review by shankari — APPROVED** [review]

**Pull request merged**